### PR TITLE
Task to check whether a number is prime

### PR DESCRIPTION
**Pull request by DariaDolia**

### CODE REVIEW

File: check_is_number_prime.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        try:
+            number = int(input('\nВведите натуральное число: '))
+            if number == 0 or number == 1:
+                print('Это число не является ни простым, ни составным')
+            elif number < 0:
+                print('Простое число может быть только натуральным.')
+                print('То есть необходимо ввести целое, положительное число.')
+            else:
+                 return number
+        except ValueError:
```
Comment (buzbohdan) [inline check_is_number_prime.py:12]: It's better to cover with `try...except` only code that can create an error, the rest of the code can be after `except`

File: check_is_number_prime.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            print('Введено некоректоное значение.')
+            print('Введите натуральное число.')
+
+
+def is_prime(number):
+    status = True    
+    for divisor in range(2, number):
+        if number % divisor == 0:
+            status = False
```
Comment (buzbohdan) [inline check_is_number_prime.py:21]: You can simplify your code by removing `status` variable completely

File: check_is_number_prime.py (hunk 1 above)
Comment (DariaDolia) [reply check_is_number_prime.py:12]: I did that, but it worked bad. If except instruction worked out the progtam return me None and mistake after that.

**Review by buzbohdan — APPROVED** [review]
Looks great!

**Pull request merged**